src/components/checkboxes/checkboxes.yaml

```diff
@@ -43,6 +43,32 @@ examples:
         value: irish
         text: Irish
 
+- name: with attributes on items
```

Comment:
Not documented, we are trying to only add .yaml examples where we cannot write a unit test for it.

In this case could you remove this example and test that this functionality works by adding a test in `checkboxes/template.test.js` please?

Reply:
@nickcolley Ah that's fine, makes sense. Added two new tests!